assets/dev/js/editor/document/commands/base/history/debounce.js

```diff
@@ -45,11 +47,38 @@ export default class Debounce extends History {
 		}
 	}
 
+	addTransaction() {
+		if ( Debounce.isTransactionStarted ) {
```

Comment:
It's  a bug fix?

Reply:
no, just better way of interacting with debounce based commands. provide us way to free some objects:
```
		if ( Debounce.isTransactionStarted ) {
			delete this.history.title;
			delete this.history.subTitle;
		}
```